api/python/t4/__init__.py

```diff
@@ -12,7 +12,8 @@
     list_packages,
     search,
     config,
-    delete_package
+    delete_package,
+    login_user_pass
```

Comment:
does this need to import all the `role` endpoints you just added

Reply:
they hang off `t4.admin` now